tests/test_lag_2.py

```diff
@@ -0,0 +1,128 @@
+    # TODO: convert add_container_to_inventory.yml
+
+    mg_facts = duthost.minigraph_facts(host=hostname)['ansible_facts']
+    fanout_neighbors = conn_graph_facts['device_conn']
+    vm_neighbors = mg_facts['minigraph_neighbors']
+
+    # Copy PTF test into PTF-docker for test LACP DU
+    test_files = ['lag_test.py', 'acs_base_test.py', 'router_utils.py']
+    for test_file in test_files:
+        src = "roles/test/files/acstests/%s" % test_file
+        dst = "/tmp/%s" % test_file
+        ptfhost.copy(src=src, dest=dst)
+    
+    # Copy tests to the PTF-docker
+    ptfhost.copy(src="ptftests", dest="/root")
+
+    # Inlucde testbed topology configuration
+    if testbed_type == 't1-lag':
+        # TODO
+    elif testbed_type == 't0':
+        # TODO
+    elif testbed_type == 't0-116'
+        # TODO
+
+    dut_mac = duthost['ansible_Ethernet0']['macaddress']
+
+    # Test each lag interface
+    for lag_name in lag_facts['names']
+        yield ptfhost, fanout_neighbors, vm_neighbors, mg_facts, lag_facts, lag_name
+
+def test_single_lag_lacp_rate(common_setup_teardown, testbed_devices):
+    ptfhost, fanout_neighbors, vm_neighbors, mg_facts, lag_facts, lag_name = common_setup_teardown
+
+    po = lag_name
+    po_interfaces = lag_facts[lag_name]['po_config']['ports']
+    po_intf_num = len(po_interfaces)
+    po_min_links = lag_facts[lag_name]['po_config']['runner']['min_ports']
+
+    # Pick flap interface name and calculate when it flaps, should portchannel interface flap or not
+    po_flap = float(po_intf_num - 1) / po_min_links * 100 < 75
+    flap_intf = lag_facts[lag_name]['po_config']['ports'].keys()[0]
+    if not po_flap:
+        pytest.skip("The interface [%s] is not flap" % flap_intf)
+
+    # Figure out fanout switches info for the flapping lag member
+    peer_device = fanout_neighbors[flap_intf]['peerdevice']
+    neighbor_interface = fanout_neighbors[flap_intf]['peerport']
+    conn_graph_facts = get_conn_graph_facts(testbed_devices, host=peer_device)
+
+    # Figure out remote VM and interface info for the flapping lag member and run minlink test
+    peer_device = vm_neighbors[flap_intf]['name']
+    neighbor_interface = vm_neighbors[flap_intf]['port']
+    peer_hwsku = 'Arista-VM'
+    peer_host = mg_facts['minigraph_devices']
+
+    # Prepare for the remote VM interfaces that using PTF docker to check if the LACP DU packet rate is correct
+    iface_behind_lag_member = []
+    for minigraph_neighbor in minigraph_neighbors:
+        if peer_device == minigraph_neighbor.value.name:
+            iface_behind_lag_member.append(mg_facts['minigraph_port_indices'][minigraph_neighbor.key])
+
+    neighbor_lag_intfs = []
+    for po_interface in po_interfaces:
+        neighbor_lag_intfs.append(vm_neighbors[po_interface]['port'])
+    
+    try:
+        lag_rate_current_setting = None
+
+        # Make sure all lag members on VM are set to fast
+        # TODO: login peer_host and use action [apswitch]
```

Comment:
Not found existing example regarding ansible: action: apswitch template=
I will check ansible api document later.

I found that following command can parse the template, maybe I need to combine some commands here
```
host.templete(src='', dest='')
```

Reply:
I will try import ansible eos config module to do the same thing